src/utils/AuthContext.js

```diff
@@ -1,15 +1,25 @@
 import React, { useState, createContext } from "react";
+import axiosApi from "./AxiosApi";
 
 export const IsAuthContext = createContext({});
```

Comment:
This is a weird variable name. 
IS this the Auth Context? Answer: empty object 🤷 

Only prefix the variable name with `is` if it's a boolean so you can read it like this:
e.g.
```javascript
const age = 12;
const isCorrectAge = age === 12 // which will yield to a "true" or "false" value (boolean)
```